### PR TITLE
remove setting address while a new customer/vendor/employee or import a new customer

### PR DESCRIPTION
**Pull request by TianXing-Yi**
This will be more efficient.

I find a small problem, in Employee Report, the name appears twice.

### CODE REVIEW

File: gnucash/gnome/dialog-vendor.c [hunk 1]
```diff
@@ -211,23 +211,10 @@ gnc_vendor_window_ok_cb (GtkWidget *widget, gpointer data)
 
     /* Check for valid company name */
     if (check_entry_nonempty (vw->company_entry,
-                              _("You must enter a company name. "
-                                "If this vendor is an individual (and not a company) "
-                                "you should enter the same value for:\nIdentification "
-                                "- Company Name, and\nPayment Address - Name.")))
+                              _("The Company Name field cannot be left blank,\n"
+                                "enter a company name or personal.")))
```
Comment (fellen) [inline gnucash/gnome/dialog-vendor.c:215]: I do not understand the meaning of `or personal`.

File: gnucash/gnome/dialog-vendor.c [hunk 2]
```diff
@@ -211,23 +211,10 @@ gnc_vendor_window_ok_cb (GtkWidget *widget, gpointer data)
 
     /* Check for valid company name */
     if (check_entry_nonempty (vw->company_entry,
-                              _("You must enter a company name. "
-                                "If this vendor is an individual (and not a company) "
-                                "you should enter the same value for:\nIdentification "
-                                "- Company Name, and\nPayment Address - Name.")))
+                              _("The Company Name field cannot be left blank,\n"
```
Comment (fellen) [inline gnucash/gnome/dialog-vendor.c:214]: What is the purpose of the hard linebreak here?

File: gnucash/gnome/dialog-vendor.c (hunk 1 above)
Comment (TianXing-Yi) [reply gnucash/gnome/dialog-vendor.c:215]: Hi, sir
I mean "a company name or a person's name", Maybe I use a wrong word.

File: gnucash/gnome/dialog-vendor.c (hunk 2 above)
Comment (TianXing-Yi) [reply gnucash/gnome/dialog-vendor.c:214]: Just keep the length of line same as before.
Now I find it's unnecessary.

how about this:
"The Company Name field cannot be left blank, please "
 "enter a company name or a person's name."

Should I close this PR , and creat a new one?

Comment (jralls) [reply gnucash/gnome/dialog-vendor.c]: No need for a new PR, just amend the commit and force push:
```
git commit -a --amend
git push -f my-repo maint
```
(substituting your actual remote for 'my-repo' of course).

File: gnucash/gnome/dialog-vendor.c (hunk 1 above)
Comment (jralls) [reply gnucash/gnome/dialog-vendor.c]: Right words, wrong order. "enter a company or personal name."  Your proposal below is also fine, or you could just leave that part out. It doesn't really matter what the user puts in the Company Name field as long as it's meaningful to them. "Cash Sale" used to be common for retail sales to walk-in customers before retailers figured out that they could make money by extracting customer's names and addresses and selling the info.

Comment (TianXing-Yi) [reply gnucash/gnome/dialog-vendor.c]: Yeah sir, Get it. Thanks for your explanation.

File: gnucash/gnome/dialog-vendor.c (hunk 2 above)
Comment (TianXing-Yi) [reply gnucash/gnome/dialog-vendor.c]: Thanks for your patient guidance. 
I have added another commit.

**Comment by jralls** [comment]
Thanks!

**Comment by fellen** [comment]
@TianXing-Yi Can we get a PR for at least the english parts of gnucash-docs reflecting this change, too?

**Comment by TianXing-Yi** [comment]
> @TianXing-Yi Can we get a PR for at least the english parts of gnucash-docs reflecting this change, too?

Ok，I will add it as soon as possible.

**Pull request merged**